recipes/cp2k/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "cp2k" %}
+{% set version = "6.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: d7dd5f164e1e51d2dcb8c7d927b99f6ac1d0f8de4a665bd9daee1a14864c30ae
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - make
+    - autoconf
+    - automake
+    - cmake 
+    - libtool
+    - python 
+  host:
+    - libblas
+    - libcblas
+    - liblapack
+    - liblapacke
```

Comment:
Can you remove `liblapacke` and `libcblas`?

Reply:
I opened a pull request to remove those https://github.com/conda-forge/cp2k-feedstock/pull/2